


from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_centralWidget(object):
    def setupUi(self, centralWidget):
        centralWidget.setObjectName("centralWidget")
        centralWidget.resize(880, 410)
        self.horizontalLayout = QtWidgets.QHBoxLayout(centralWidget)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.frame_2 = QtWidgets.QFrame(centralWidget)
        self.frame_2.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.frame_2.setObjectName("frame_2")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.frame_2)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.labelImage = QtWidgets.QLabel(self.frame_2)
        self.labelImage.setText("")
        self.labelImage.setObjectName("labelImage")
        self.horizontalLayout_2.addWidget(self.labelImage)
        self.horizontalLayout.addWidget(self.frame_2)
        self.frame = QtWidgets.QFrame(centralWidget)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame.sizePolicy().hasHeightForWidth())
        self.frame.setSizePolicy(sizePolicy)
        self.frame.setMinimumSize(QtCore.QSize(280, 0))
        self.frame.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.frame.setObjectName("frame")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.frame)
        self.verticalLayout.setObjectName("verticalLayout")
        self.frame1 = QtWidgets.QFrame(self.frame)
        self.frame1.setEnabled(True)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.frame1.sizePolicy().hasHeightForWidth())
        self.frame1.setSizePolicy(sizePolicy)
        self.frame1.setMinimumSize(QtCore.QSize(0, 100))
        self.frame1.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame1.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame1.setObjectName("frame1")
        self.gridLayout = QtWidgets.QGridLayout(self.frame1)
        self.gridLayout.setObjectName("gridLayout")
        self.label = QtWidgets.QLabel(self.frame1)
        self.label.setObjectName("label")
        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
        self.label_2 = QtWidgets.QLabel(self.frame1)
        self.label_2.setObjectName("label_2")
        self.gridLayout.addWidget(self.label_2, 1, 0, 1, 1)
        self.label_5 = QtWidgets.QLabel(self.frame1)
        self.label_5.setObjectName("label_5")
        self.gridLayout.addWidget(self.label_5, 2, 0, 1, 1)
        self.pushButtonSubmit = QtWidgets.QPushButton(self.frame1)
        self.pushButtonSubmit.setObjectName("pushButtonSubmit")
        self.gridLayout.addWidget(self.pushButtonSubmit, 3, 2, 1, 1)
        self.doubleSpinBoxTime = QtWidgets.QDoubleSpinBox(self.frame1)
        self.doubleSpinBoxTime.setMinimumSize(QtCore.QSize(124, 0))
        self.doubleSpinBoxTime.setDecimals(1)
        self.doubleSpinBoxTime.setMinimum(0.1)
        self.doubleSpinBoxTime.setMaximum(1.0)
        self.doubleSpinBoxTime.setSingleStep(0.1)
        self.doubleSpinBoxTime.setObjectName("doubleSpinBoxTime")
        self.gridLayout.addWidget(self.doubleSpinBoxTime, 0, 2, 1, 1)
        self.doubleSpinBoxStoracy = QtWidgets.QDoubleSpinBox(self.frame1)
        self.doubleSpinBoxStoracy.setDecimals(1)
        self.doubleSpinBoxStoracy.setMinimum(0.1)
        self.doubleSpinBoxStoracy.setMaximum(1.0)
        self.doubleSpinBoxStoracy.setSingleStep(0.1)
        self.doubleSpinBoxStoracy.setObjectName("doubleSpinBoxStoracy")
        self.gridLayout.addWidget(self.doubleSpinBoxStoracy, 1, 2, 1, 1)
        self.doubleSpinBoxAccuracy = QtWidgets.QDoubleSpinBox(self.frame1)
        self.doubleSpinBoxAccuracy.setDecimals(1)
        self.doubleSpinBoxAccuracy.setMinimum(0.1)
        self.doubleSpinBoxAccuracy.setMaximum(1.0)
        self.doubleSpinBoxAccuracy.setSingleStep(0.1)
        self.doubleSpinBoxAccuracy.setObjectName("doubleSpinBoxAccuracy")
        self.gridLayout.addWidget(self.doubleSpinBoxAccuracy, 2, 2, 1, 1)
        self.verticalLayout.addWidget(self.frame1)
        self.frame_21 = QtWidgets.QFrame(self.frame)
        self.frame_21.setFrameShape(QtWidgets.QFrame.Panel)
        self.frame_21.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_21.setObjectName("frame_21")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.frame_21)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.horizontalLayout_6 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_6.setObjectName("horizontalLayout_6")
        self.pushButtonOpen = QtWidgets.QPushButton(self.frame_21)
        self.pushButtonOpen.setObjectName("pushButtonOpen")
        self.horizontalLayout_6.addWidget(self.pushButtonOpen)
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_6.addItem(spacerItem)
        self.pushButtonRun = QtWidgets.QPushButton(self.frame_21)
        self.pushButtonRun.setObjectName("pushButtonRun")
        self.horizontalLayout_6.addWidget(self.pushButtonRun)
        self.verticalLayout_2.addLayout(self.horizontalLayout_6)
        self.verticalLayout.addWidget(self.frame_21)
        self.horizontalLayout.addWidget(self.frame)
        self.horizontalLayout.setStretch(0, 5)
        self.horizontalLayout.setStretch(1, 2)

        self.retranslateUi(centralWidget)
        QtCore.QMetaObject.connectSlotsByName(centralWidget)

    def retranslateUi(self, centralWidget):
        _translate = QtCore.QCoreApplication.translate
        centralWidget.setWindowTitle(_translate("centralWidget", "MainWindow"))
        self.label.setText(_translate("centralWidget", "TimeWeight"))
        self.label_2.setText(_translate("centralWidget", "StorageWeight"))
        self.label_5.setText(_translate("centralWidget", "AccuracyWeight"))
        self.pushButtonSubmit.setText(_translate("centralWidget", "Submit"))
        self.pushButtonOpen.setText(_translate("centralWidget", "Open"))
        self.pushButtonRun.setText(_translate("centralWidget", "Run"))
